### DISCUSSION TITLE
DatePicker styling

### DISCUSSION THREAD

**Discussion by harley-hutchinson** [heroui-inc/heroui#3202] [Help]
Hi all, I am struggling with styling the new DatePicker , I want to change the text colour on the input and the picker itself.

Thanks

**Comment by phantodev** [comment]
There are a bug in this componet... The classeNames for inputwrapper is not working... To change text color or color of picker you can use this logic directly in CSS

.bg-background-date [data-slot="base"] [data-slot="input-wrapper"] {
  @apply bg-white border dark:bg-background dark:border-dark dark:focus-within:!bg-dark;
}

In my case i am creating a bg-background-date thant change things in input-wrapper slot... works perfectly...

**Reply by LucasBPPereira** [reply]
I needed to change the color of the days in the calendar modal, and apparently using it directly in the CSS worked! thanks for tip.

`[data-slot="cell"] span[role="button"] {
  @apply text-default-500 data-[hover=true]:text-primary-400 data-[selected=true]:text-primary-foreground data-[disabled=true]:text-default-300;
}`
